examples/ens/package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Tool long, you can use `example-mock-contract` as a start, note: no need to have `build` at all